src/common/options/global.yaml.in

```diff
@@ -6086,4 +6086,31 @@ options:
   services:
   - rgw
   - osd
-  with_legacy: true
\ No newline at end of file
+  with_legacy: true
+- name: mgr_inject
+  type: bool
+  level: dev
+  desc: Enable/disable injection of fake data in the mgr module
+  default: false
+  flags:
+  - runtime
+  with_legacy: true
+  services:
+  - mgr
+- name: mgr_inject_num_osds
+  type: int
+  level: dev
+  desc: Number of osds to inject in the mgr module
+  default: 0
+  flags:
+  - runtime
+  with_legacy: true
+  services:
+  - mgr
```

Comment:
@pereman2 we talked about not merging the fake OSD injector?

Reply:
yes, it should be removed